deployment_hooks/lib/LoadDeploymentActions.js

```diff
@@ -32,9 +32,9 @@ function loadActionScripts(actions) {
   _.forEach(actions, (template, actionScriptName) => {
```

Comment:
Yes. We had put this logic to avoid rewriting files on start, as it would have led to increase in monit start time. We can remove this logic as action files won't be very large.

Reply:
Done!